getstats-mangle.js

```diff
@@ -13,6 +13,7 @@ module.exports = function(stats) {
   // taken from https://github.com/fippo/adapter/tree/getstats-mangling
   const standardReport = {};
   Object.keys(stats).forEach(id => {
+    if (id === 'timestamp') return;
```

Comment:
i see braces in single line ifs in other places of the file.   It looks to me like the most common convention